src/components/Navigation/TopBar/Help.tsx

```diff
@@ -26,70 +26,95 @@ const MenuLinkItem = styled.a`
+    const closeMenu = () => setIsOpen(false);
+    const toggleMenu = () => setIsOpen(!isOpen);
 
-  return (
-    <>
-      <Button
-        variant="ghost_icon"
-        ref={buttonRef}
-        id="anchor-match"
-        aria-haspopup="true"
-        aria-expanded={isOpen}
-        aria-controls="menu-match"
-        onClick={toggleMenu}
-      >
-        <Icon
-          data={help_outline}
-          color={colors.interactive.primary__resting.hsla}
-        />
-      </Button>
-
-      <TopBarMenu
-        open={isOpen}
-        title="Help"
-        onClose={closeMenu}
-        anchorEl={buttonRef.current}
-      >
-        <MenuLinkItem
-          as="a"
-          onClick={closeMenu}
-          href={`https://amplify.equinor.com/releasenotes?app=%5B"${applicationName}"%5D`}
-          target="_blank"
+    return (
+      <>
+        <Button
+          variant="ghost_icon"
+          ref={buttonRef}
+          id="anchor-match"
+          aria-haspopup="true"
+          aria-expanded={isOpen}
+          aria-controls="menu-match"
+          onClick={toggleMenu}
         >
-          <ContentInfo>
-            <Icon
-              data={file_description}
-              size={24}
-              color={colors.interactive.primary__resting.hsla}
-            />
-            <Typography group="navigation" variant="menu_title" as="span">
-              Release notes
-            </Typography>
-          </ContentInfo>
           <Icon
-            data={external_link}
-            size={24}
+            data={help_outline}
             color={colors.interactive.primary__resting.hsla}
           />
-        </MenuLinkItem>
-      </TopBarMenu>
-    </>
-  );
-};
+        </Button>
+
+        <TopBarMenu
+          open={isOpen}
+          title="Help"
+          onClose={closeMenu}
+          anchorEl={buttonRef.current}
+          ref={ref}
+        >
+          {!hideDefaultMenuItems && (
+            <MenuLinkItem
+              as="a"
+              onClick={closeMenu}
+              href={`https://amplify.equinor.com/releasenotes?app=%5B"${applicationName}"%5D`}
+              target="_blank"
+            >
+              <ContentInfo>
+                <Icon
+                  data={file_description}
+                  size={24}
+                  color={colors.interactive.primary__resting.hsla}
+                />
+                <Typography group="navigation" variant="menu_title" as="span">
+                  Release notes
+                </Typography>
+              </ContentInfo>
+              <Spacer />
+              <Icon
+                data={external_link}
+                size={24}
+                color={colors.interactive.primary__resting.hsla}
+              />
+            </MenuLinkItem>
+          )}
+          {children && !hideDefaultMenuItems && (
+            <Divider style={{ margin: 0 }} />
+          )}
+          {children && (
+            <ContentWrapper onClick={() => closeMenu()}>
```

Comment:
This can be simplified to just:
```suggestion
            <ContentWrapper onClick={closeMenu}>
```